lib/units/ios-device/plugins/screen/stream.js

```diff
@@ -101,7 +101,7 @@ module.exports = syrup.serial()
               log.important('ws on close event')
             }
             
-            if (options.deviceType === 'tvOS' || orientation === 'PORTRAIT') {
+            if (WdaClient.deviceType === 'Apple TV' || orientation === 'PORTRAIT') {
```

Comment:
any risk with this? I suppose all existing `Apple TV` were registered using tvOS...

Reply:
No problems, since this will be checked after using `startSession`, the device type will already be set. 

I checked every exceptional case for AppleTV with this new property